airbyte-integrations/connectors/source-faker/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_faker_singer'
```

Comment:
this should be `source_faker` right?

Reply:
I... guess?  That's what was generated. I'll update it (9c1b37a5b8a4d2be1ed42a52f113b0be957ad0f7)